azure-pipelines-official.yml

```
@@ -82,7 +82,7 @@ stages:
     displayName: Official Build
     timeoutInMinutes: 360
     pool:
-      name: NetCore1ESPool-Svc-Internal
+      name: NetCore1ESPool-Internal
       demands: ImageOverride -equals Build.Windows.Amd64.VS2022.Pre
```

Until certs are available.

```suggestion
      demands: ImageOverride -equals Build.Server.Amd64.VS2019
```